python/pip_install/pip_repository.bzl

```diff
@@ -775,4 +823,4 @@ def _repr_dict(items):
     return {k: repr(v) for k, v in items.items()}
 
 def _format_dict(items):
-    return "{{{}}}".format(", ".join(sorted(['"{}": {}'.format(k, v) for k, v in items.items()])))
+    return "{{\n{}}}".format("".join(sorted(['    "{}": {},\n'.format(k, v) for k, v in items.items()])))
```

Comment:
Can we use a platform independent newline?  It is a nit pick.

Reply:
Would be happy to, not sure what that would be and not finding obvious documentation hits searching

Reply:
Wound up just reverting this to avoid possible windows/bzlmod impact.